base/include/TextOverlayXForm.h

```diff
@@ -46,7 +46,6 @@ class TextOverlayXForm : public Module
 	bool validateInputPins();
 	bool validateOutputPins();
 	void addInputPin(framemetadata_sp &metadata, string &pinId);
-	void setProps(TextOverlayXForm);
```

Comment:
what is the idea here

Reply:
It was just a redundant code, We have declared but it's never defined